ingest/flow_to_s3.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
same comments as for the dns_to_s3 script.... 

the script should have a comment at top saying what it does, and expecting people to simply edit values (rather than use CLI parameters) strikes me as sketchy but maybe the bash script world is different